prow/config/jobs.go

```diff
@@ -145,6 +145,8 @@ type Postsubmit struct {
 	Spec *v1.PodSpec `json:"spec,omitempty"`
 	// Maximum number of this job running concurrently, 0 implies no limit.
 	MaxConcurrency int `json:"max_concurrency,omitempty"`
+	// Report is weather the job need to report through crier
+	Report bool `json:"report,omitempty"`
```

Comment:
our APIs are a bit awkward / inconsistent :grimacing: /shrug

Reply:
we probably should somehow flip skip_report to report and default it to true for presubmit jobs...

Reply:
That's not how go works. If you want report to default to true you create a variable named `skip_report` :stuck_out_tongue: 

It seems like this isn't needed -- just use skip_report

Reply:
we want report for periodic/postsubmit here to be default false though :-)

Reply:
How are periodic and postsubmit jobs going to report? Where are they going to report? Maybe this should be a URL or something?

Reply:
This feels like a hack and I don't think we should take this PR without further thought on the api and design.

In the mean time, can't we use annotations/labels to determine whether to report? The pubsub jobs are created with mkpj so this is something we can handle there.

Reply:
if we use annotation/labels for other reporters, why github jobs need to be special and have first class report boolean?

I'm ok with in the future we switch github reporter also respect annotation/labels, and drop the `Report` field.

Reply:
switched to https://github.com/kubernetes/test-infra/pull/9858

as the goal is simply allow periodic pubsub job to report